sdl_viewer/src/main.rs

```diff
@@ -215,13 +217,14 @@ impl NodeView {
             _buffer_position: buffer_position,
             _buffer_color: buffer_color,
             meta: node_data.meta,
+            used_memory: position.len() + color.len(),
         }
     }
 }
 
 // Keeps track of the nodes that were requested in-order and loads then one by one on request.
 struct NodeViewContainer {
-    node_views: HashMap<octree::NodeId, NodeView>,
+    cache: LruCache<octree::NodeId, NodeView>,
```

Comment:
why did you rename this to cache? It still is conceptually the `node_views` hashmap